scrapers/joymii.yml

```diff
@@ -0,0 +1,34 @@
+name: "joymii"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - joymii.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="info col-xs-12"]
+    scene:
+      Title: $info//h1[@class="title"]/text()
+      Details: $info//p/text()
+      Date:
+        selector: $info//h1[@class="title"]/text()
+        replace:
+          - regex: ^
+            with: "https://joymii.com/search?query="
+          - regex: \s
+            with: +
+        subScraper:
+          selector: //div[@class="box-results row"]/div[1]//div[@class="box-release_date release_date"]/text()
+        parseDate: Jan 02, 2006
+      Image: //video/@poster
```

Comment:
in some scenes the video poster is missing
`https://joymii.com/site/set-photo/code/6744323_jmu992`
`https://joymii.com/site/set-photo/code/20795056_ywq526`
`https://joymii.com/site/set-photo/code/20785844_clk308`
The below seems to work 
```yaml
Image: //img[@class="cover"]/@src|//video/@poster
```
It might need some more testing though

Reply:
Seems to work in all the tests I tried. Updated